k8s/production/runners/signing/release.yaml

```diff
@@ -151,6 +151,11 @@ spec:
               "metrics/gitlab_ci_project_name" = "$CI_PROJECT_NAME"
               "metrics/gitlab_ci_job_stage" = "$CI_JOB_STAGE"
               "metrics/gitlab_ci_commit_ref_name" = "$CI_COMMIT_REF_NAME"
+              "metrics/spack_job_spec_pkg_version" = "$SPACK_JOB_SPEC_PKG_VERSION"
+              "metrics/spack_job_spec_compiler_name" = "$SPACK_JOB_SPEC_COMPILER_NAME"
+              "metrics/spack_job_spec_compiler_version" = "$SPACK_JOB_SPEC_COMPILER_VERSION"
+              "metrics/spack_job_spec_arch" = "$SPACK_JOB_SPEC_ARCH"
+              "metrics/spack_job_spec_variants" = "$SPACK_JOB_SPEC_VARIANTS"
```

Comment:
These variables aren't applicable to this particular job, it runs at the end of each child pipeline and operates on all the specs in the mirror.